tensorrt_llm/serve/openai_disagg_server.py

```diff
@@ -128,6 +128,7 @@ async def merge_streaming_responses(self, ctx_response,
     async def openai_completion(self, req: CompletionRequest) -> Response:
         try:
             gen_req = copy.deepcopy(req)
+            print("************** Receive request **************")
```

Comment:
Please cleanup the prints, and for necessary debugging info, it is suggested to use `logger.info/debug` instead.